CMakeLists.txt

```diff
@@ -66,6 +66,7 @@ OPTION(USE_OPENCL "Use OpenCL" OFF)
 OPTION(ENABLE_DIST "Enable distributed training" OFF)
 OPTION(DISABLE_WARNINGS "Disable warnings under windows" ON)
 OPTION(USE_MODULES "Compile dependent libs as submodules together with singa" OFF)
+OPTION(USE_MKLDNN "Use mkl-dnn libs" ON)
```

Comment:
This line will make USE_MKLDNN enabled by default. Is this the required default configuration?

Also please remember to add this new option to the[ installation documentation](https://github.com/apache/incubator-singa/blob/master/doc/en/docs/installation.md).

Reply:
Hi Moaz, Thank you for the advice, this is resolved.